packages/core/src/rules/cognitoEnforceLongPasswords/doc.md

```diff
@@ -0,0 +1,44 @@
+# User Pools should enforce passwords of at least 10 characters
+
+## Why is this important?
+
+By default, Cognito User Pools enforce passwords of at least 8 characters. This is a weak password policy and should be changed to at least 10 characters. Passwords of 8 characters are quite easy to guess and brute force.
+
+## How to fix
+
+To comply, set the `MinimumPasswordLength` property of your userPools to `10` or higher.
```

Comment:
tu parles de la console là? si c'est le cas tu peux préciser et dire qu'il faut regarder l'onglet 'sign in experience' de ton userpool
si tu parles d'autre chose, c'est pas clair 🙈